test/system/inspect.test.js

```diff
@@ -909,6 +909,66 @@ const pipenvAppExpectedDependencies = {
   },
 };
 
+test('inspect Pipfile in nested directory', (t) => {
```

Comment:
How much work will it be to have this test in the jest tests? (`inspect.spec.ts`)

Reply:
Not familiar with it but shouldnt be too hard :)

Reply:
Not implementing as to get ticket out for customer